### ISSUE TITLE
I cannot get my project to preview properly anymore

### ISSUE THREAD

**Issue by ruudio** [stackblitz/bolt.new#4378]
### Describe the bug

I loved how it put in all functionality I requested, then bolt ran into problems around icons etc. and could not preview the page anymore. I wasted at least 5 million tokens and 4 hours of time to fix this, and I am nowhere near completion. Please fix this bug, so I can preview and finalize the project

### Link to the Bolt URL that caused the error

https://bolt.new/~/sb1-9rartpt1

### Validations

- [X] Please make your project public or accessible by URL. This will allow anyone trying to help you to easily reproduce the issue and provide assistance.

### Steps to reproduce

Preview the the site

### Expected behavior

The site does not preview, throws errors for no reason

### Platform

```
Browser name = Firefox
Full version = 133.0
Major version = 133
navigator.appName = Netscape
navigator.userAgent = Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
performance.memory = undefined
Username = rudigermerz
Chat ID = e9064bc1065a
```

**Comment by ruudio** [comment]
There seems to be a problem with the social icons that Bolt cannot get passed

**Comment by Labhk** [comment]
I was able to open this in stackblitz I can also see the preview
![image](https://github.com/user-attachments/assets/b10a9f80-96a0-444c-b468-3782f1d28bf3)

**Comment by ruudio** [comment]
Yes, after I worked on it tirelessly, after many mistakes and burning my
tokens, please have a look at the chat history. It has been like a house
cards, one prompt and the whole falls apart as the project grows. I went
correcting errors from from about 7 million tokens to 1.8 million. Can I
get these back?

**Comment by Labhk** [comment]
you can write a email to hello@stackblitz.com, explain the issue.

**Comment by ruudio** [comment]
Thank you, I will do that.

**Comment by ruudio** [comment]
Hi can you have a look please? I am stuck

**Comment by Labhk** [comment]
Is it the same project? I just opened the above porject in stackblitz
it's working..
![Uploading image.png…]()

**Comment by Labhk** [comment]
> Is it the same project? I just opened the above project in stackblitz it's working..
but In bolt preview I see the white preview, you can also try opening in stackblitz and run it there.

**Comment by ruudio** [comment]
Sorry, it a different link, I posted it on github. It is this project -
thank for looking: https://bolt.new/~/sb1-raq3rf

https://github.com/stackblitz/bolt.new/issues/4734

Project not in working state: plugin:vite:esbuild issues #4734

**Comment by ruudio** [comment]
Worked on it it tirelessly, to no avail.

**Comment by Labhk** [comment]
https://sb1-xmwzkdsf.stackblitz.io

fixed here use this,
btw good design

**Comment by ruudio** [comment]
Awesome, thanks! Site is not running well.

How do I use this?  https://sb1-xmwzkdsf.stackblitz.io

[image: image.png]

**Comment by ruudio** [comment]
[image: image.png]

**Comment by Labhk** [comment]
https://stackblitz.com/edit/sb1-xmwzkdsf
